.github/workflows/translations.yml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
+
+name: Update Translations
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
```

Comment:
This will run on the first day of every month at 12 a.m.
Is is too often? Or too rare? Suggestions welcome.

Reply:
I would almost vote too rare - how expansive is a run? (build time, bandwidth, ...)

Reply:
For main the job takes 51 secs on my fork, see https://github.com/Holzhaus/mixxx/actions/runs/11354842443/job/31582931642

Reply:
I have the experience that the connections sometimes stales at a certain percentage and continues after a couple of minutes.

Reply:
We could do it weekly. The issue would fix itself after one week.